### ISSUE TITLE
When I try to package in 5.1, it fails.

### ISSUE THREAD

**Issue by thegreatsai** [endink/Mediapipe4u-plugin#44]
When I try to package an exe with my custom model or even the included model, after a while I get the error below. Could you help me out with it. I have seen the error in 5.0 as well. 

**### Packaging (Windows): Missing precompiled manifest for 'MediaPipe', 'D:\MoCap\MediaPipe4U-Demo-main-March2023\MediaPipe4U-Demo-main\Plugins\MediaPipe4U\Intermediate\Build\Win64\UnrealGame\Development\MediaPipe\MediaPipe.precompiled'. This module was most likely not flagged for being included in a precompiled build - set 'PrecompileForTargets = PrecompileTargetsType.Any;' in MediaPipe.build.cs to override. If part of a plugin, also check if its 'Type' is correct.
UATHelper: Packaging (Windows): Took 12.569327600000001s to run dotnet.exe, ExitCode=6
UATHelper: Packaging (Windows): UnrealBuildTool failed. See log for more details. (C:\Users\Opu\AppData\Roaming\Unreal Engine\AutomationTool\Logs\D+Games+UE_5.1\UBT-MediaPipe4U_Demo-Win64-Development.txt)
UATHelper: Packaging (Windows): AutomationTool executed for 0h 0m 17s
UATHelper: Packaging (Windows): AutomationTool exiting with ExitCode=6 (6)
UATHelper: Packaging (Windows): BUILD FAILED
PackagingResults: Error: Unknown Error**

**Comment by endink** [comment]
Why not use the issue template, fill in the necessary information？

**Comment by endink** [comment]
https://opensource-labijie-com.translate.goog/Mediapipe4u-plugin/package/source_engine_package.html?_x_tr_sl=zh-CN&_x_tr_tl=en&_x_tr_hl=zh-CN&_x_tr_pto=wapp

**Comment by AOK-VISIONS** [comment]
Hi there, I'm actually having the exact same issue here. I've followed the documentation over and over. 
Everything matches and I've followed the steps, however persistently receive the same error as above.

Before running the build everything is setup correctly, and "MediaPipe.precompiled" appears in the correct place.
BUT upon running the build:
The "MediaPipe.precompiled" file, including its sub heading folders: "MediaPipe" and "Development/Shipping" and adjacent files, all seem to delete and disappear once the build reaches this error.

I would also appreciate if there is any further help with this? Thank you.
